RELEASE/scripts/autoscend.ash

```diff
@@ -10390,6 +10421,7 @@ boolean L4_batCave()
 	if((item_amount($item[Sonar-In-A-Biscuit]) > 0) && (batStatus < 3))
 	{
 		use(1, $item[Sonar-In-A-Biscuit]);
+		visit_url("place.php?whichplace=bathole"); // ensure quest status is updated.
```

Comment:
I added this as I had a run where it kept spamming The Batrat and Ratbat Burrow and using a sonar-in-a-biscuit after every adventure (which does nothing) because questL04Bat didn't get updated to step3 the first time it used a sonar-in-a-biscuit probably due to a timeout.